backend/server/routers/user.py

```diff
@@ -99,6 +99,10 @@ def default_cs_user() -> Storage:
 def toggle_summer_term(token: str = DUMMY_TOKEN):
     user = get_user(token)
     user['planner']['isSummerEnabled'] = not user['planner']['isSummerEnabled']
+    if not user['planner']['isSummerEnabled']:
```

Comment:
There doesn't appear to be sample data for this in `example_local_storage_data.json` as no data in that file has a populated `T0`. What would you like me to do?

Reply:
Fixed